include/aquamarine/allocator/GBM.hpp

```diff
@@ -53,17 +82,18 @@ namespace Aquamarine {
         CGBMAllocator(int fd_, Hyprutils::Memory::CWeakPointer<CBackend> backend_);
 
         // a vector purely for tracking (debugging) the buffers and nothing more
-        std::vector<Hyprutils::Memory::CWeakPointer<CGBMBuffer>> buffers;
+        std::vector<Hyprutils::Memory::CWeakPointer<IBuffer>> buffers;
```

Comment:
as before, GBM allocator should only allocate GBM. for dumb, please make a dumb allocator.